### ISSUE TITLE
Fails in pytest==4.1.0

### ISSUE THREAD

**Issue by monkeysecurity** [reverbc/pytest-concurrent#29]
Works for me on `pytest==4.0.2` but fails on `pytest==4.1.0`

Works on:
```
platform linux -- Python 3.7.1, pytest-4.0.2, py-1.7.0, pluggy-0.8.1
cachedir: .tox/py37/.pytest_cache
rootdir: XXXX, inifile:
plugins: cov-2.6.1, concurrent-0.1.2
collected 43 items
```

Fails on:
```
platform linux -- Python 3.7.1, pytest-4.1.0, py-1.7.0, pluggy-0.8.1
cachedir: .tox/py37/.pytest_cache
rootdir: XXXX, inifile:
plugins: cov-2.6.1, concurrent-0.1.2
collected 43 items
```

Error is:
```
 Traceback (most recent call last):
   File "MYPATH/.tox/py37/lib/python3.7/site-packages/_pytest/main.py", line 203, in wrap_session
     session.exitstatus = doit(config, session) or 0
   File "MYPATH/.tox/py37/lib/python3.7/site-packages/_pytest/main.py", line 243, in _main
     config.hook.pytest_runtestloop(session=session)
   File "MYPATH/.tox/py37/lib/python3.7/site-packages/pluggy/hooks.py", line 284, in __call__
     return self._hookexec(self, self.get_hookimpls(), kwargs)
   File "MYPATH/.tox/py37/lib/python3.7/site-packages/pluggy/manager.py", line 68, in _hookexec
     return self._inner_hookexec(hook, methods, kwargs)
   File "MYPATH/.tox/py37/lib/python3.7/site-packages/pluggy/manager.py", line 62, in <lambda>
     firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
   File "MYPATH/.tox/py37/lib/python3.7/site-packages/pluggy/callers.py", line 208, in _multicall
     return outcome.get_result()
   File "MYPATH/.tox/py37/lib/python3.7/site-packages/pluggy/callers.py", line 80, in get_result
     raise ex[1].with_traceback(ex[2])
   File "MYPATH/.tox/py37/lib/python3.7/site-packages/pluggy/callers.py", line 187, in _multicall
     res = hook_impl.function(*args)
   File "MYPATH/.tox/py37/lib/python3.7/site-packages/pytest_concurrent.py", line 104, in pytest_runtestloop
     concurrent_group_marker = item.get_marker('concgroup')
 AttributeError: 'Function' object has no attribute 'get_marker'
```

**Comment by monkeysecurity** [comment]
Good find @mikegrima - I just patched that locally and it worked.

Let me see if I can make a PR real quick.

**Comment by reverbc** [comment]
@monkeysecurity Thanks for verifying!

I believe you are talking about https://travis-ci.org/reverbc/pytest-concurrent/builds/478876830

For py36, it's actually failing because it didn't pass `flaske8` only we use py36 as executor hence the confusion. I fixed this issue with https://github.com/reverbc/pytest-concurrent/commit/469eebcdb728e2e32f7a9b4eb91bd044ebc90be1

For py27, I don't have a solid idea at the moment. But since py27 is going away soon, I decided to remove py2 support and put our effort in py3+. Hope this decision won't impact you.

I'm closing this issue now as the original issue was fixed and verified, but welcome to continue the discussion below or if necessary, open a new issue. Thanks again for the fix and most importantly, using this library :)
